ipfshttpclient/client/base.py

```diff
@@ -9,11 +9,14 @@
 from .. import multipart, http, utils
 
 
+#TODO: Review if this is still necessary after requiring Python 3.7.
```

Comment:
Change this to *#XXX: Review if this is still necessary after requiring Python 3.8 which introduced \`Protocol\`*

`Protocol` [was only introduced in 3.8](https://docs.python.org/3.9/library/typing.html#typing.Protocol) and is the reason for this hack

Reply:
I can change the comment, but I'm not sure what you want for `XXX` - can you clarify?